### ISSUE TITLE
port to STM32F103C8

### ISSUE THREAD

**Issue by JBSchueler** [deadsy/grbl_stm32f4#4]
Nowadays there are very nice (and cheap) development boards for the STM32F103C8 like [this](https://www.youtube.com/watch?v=Ze6q6NidS5w) one.

How easy/difficult is it to port this code to the STM32F103C8?
Is the code fully functional for a STM32?
Especially GRBL 0.9j?

**Comment by deadsy** [comment]
Yes - you'll have to remap the IO to whatever the available pins are on your board. The core drivers should be OK since you are on an ST chip and the peripherals don't change much, but you'll have to make sure the base addresses are ok.

Later versions of both grbl and the cube drivers are on the development branch. Most of it works but I still need to finish of the stepper motor drivers..
